.github/workflows/validation.yml

```diff
@@ -0,0 +1,57 @@
+name: Verify
+on:
+  push:
+    branches: [master]
+  pull_request:
+permissions:
+  contents: read
+defaults:
+  run:
+    shell: bash  
+concurrency:
+  group: ${{ github.head_ref || github.ref_name }}
+  cancel-in-progress: true  
+jobs:
+  verify:
+    strategy:
+      matrix:
+        os: [ubuntu, windows]
+    runs-on: ${{ matrix.os }}-latest
+    permissions:
+      # for EnricoMi/publish-unit-test-result-action
+      issues: read
+      checks: write
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+      - uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+      - name: Set TB License
+        run: |
+          TBL=${{ secrets.TB_LICENSE }}
+          [ -z "$TBL" ] && echo "No TB license provided" && exit 1
+          mkdir -p ~/.vaadin/
+          echo '{"username":"'`echo $TBL | cut -d / -f1`'","proKey":"'`echo $TBL | cut -d / -f2`'"}' > ~/.vaadin/proKey
+      - uses: actions/cache@v3
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-maven-
+      - uses: browser-actions/setup-chrome@latest
+        with:
+          chrome-version: stable
+      - uses: nanasess/setup-chromedriver@master
```

Comment:
I wonder if this is required. Shouldn't Selenium download the correct driver manager?
Or is this step here because this workflow will run with Testbench version with older selenium versions?

Reply:
good point, removed, in this we also check that driver is correctly downloaded by selenium